actions/login-to-gcs/README.md

```diff
@@ -0,0 +1,42 @@
+# login-to-gcs
+
+This is a composite GitHub Action, used to push objects to a bucket in Google Cloud Storage (GCS).
+It uses [OIDC authentication](https://docs.github.com/en/actions/deployment/security-hardening-your-deployments/about-security-hardening-with-openid-connect)
+which means that only workflows which get triggered based on certain rules can
+trigger these composite workflows.
+
+```yaml
+name: Login-to-gcs
+
+on:
+  push:
+    branches:
+      - main
+
+env:
+  ENVIRONMENT: "dev"
+
+permissions:
+  contents: read
+  id-token: write
+
+jobs:
+  login-to-gcs:
+    name: login-to-gcs
+    steps:
+      - uses: grafana/shared-workflows/actions/login-to-gcs@rwhitaker/push-to-gcs
+        id: login-to-gcs
+```
```

Comment:
Suggest adding an example of using the bucket here - maybe can show that `gcloud` cli works?

```console
$ gcloud storage cp OBJECT_LOCATION gs://DESTINATION_BUCKET_NAME
```

as in: you don't have to use the `google-github-actions/upload-cloud-storage`

Reply:
cheers - I would add that as a step in the example, but that's not required :slightly_smiling_face: